go/nbs/store.go

```diff
@@ -313,6 +315,13 @@ func (nbs *NomsBlockStore) AddHints(hints types.Hints) {
 }
 
 func (nbs *NomsBlockStore) Flush() {
-	success := nbs.UpdateRoot(nbs.root, nbs.root)
-	d.Chk.True(success)
+	b := &backoff.Backoff{
+		Min:    50 * time.Millisecond,
+		Max:    10 * time.Second,
```

Comment:
Think a lower min might make sense. Seems like we're mainly concerned with two cases:

1) Someone updated behind your back a long time ago, but isn't about to try to update any time soon. In this case *any* min will suffice, since we'll succeed on the next UpdateRoot

2) There's someone else whose about to try the same thing and we want to avoid a three-stoges routine of going exactly back & forth. I suspect that 10s of microseconds would accomplish this as long as we keep jitter.

The only downside I know of to picking too low is that we get a few first back & forths with another Flusher -- which means read load on the dynamo table.

So to counter your from air proposal, here's another: 128 microseconds.